Dockerfile.tkm-agent

```diff
@@ -0,0 +1,50 @@
+# Build the agent binary
```

Comment:
Can we call them Containerfile.x instead of Dockerfile.x? I'm not a Podman person, but you are :D

Reply:
Not a big deal, it can be done later or not at all.

Reply:
yeah I can update that